.github/workflows/continuous-integration.yml

```diff
@@ -1,19 +1,15 @@
-# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
 
 name: "Continuous Integration"
 
 on:
   pull_request:
-  push:
-    branches:
-      - "master"
 
 jobs:
 
   unit-tests:
     name: "Unit tests"
 
-    runs-on: "ubuntu-18.04"
+    runs-on: "ubuntu-latest"
```

Comment:
i find this very risky, but maybe since stuff runs inside docker, does not matter that much

Reply:
Woops I noticed this comment after merging :sweat_smile: 
Yeah I find it risky too, but a little (deps are fixed). cc @bendavies @localheinz in case you have an opinion to share on this.